<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
	background-color: rgb(2, 137, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="title">
     <item>
      <widget class="QLabel" name="icon_label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../weather.qrc">:/title/img/icon.ico</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>godbei天气</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="setting_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>设置</string>
       </property>
       <property name="styleSheet">
        <string notr="true">#setting_btn:hover{
	
	border-image: url(:/title/img/title/setting_hover.png);
}
#setting_btn:pressed{
	
	border-image: url(:/title/img/title/setting_pressed.png);
}

#setting_btn{
	
	border-image: url(:/title/img/title/setting_normal.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pin_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>置顶</string>
       </property>
       <property name="styleSheet">
        <string notr="true">#pin_btn:hover{
	
	border-image: url(:/title/img/title/pin_hover.png);
}


#pin_btn:checked{
	
	border-image: url(:/title/img/title/pin_true.png);
}



#pin_btn{
	
	border-image: url(:/title/img/title/pin_normal.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minimize_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>最小化</string>
       </property>
       <property name="styleSheet">
        <string notr="true">#minimize_btn:hover{
	
	border-image: url(:/title/img/title/minimize_icon_hover.png);
}
#minimize_btn:pressed{
	
	border-image: url(:/title/img/title/minimize_icon_pressed.png);
}

#minimize_btn{
	
	border-image: url(:/title/img/title/minimize_icon_normal.png);
	
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>关闭</string>
       </property>
       <property name="styleSheet">
        <string notr="true">#close_btn:hover{
	
	border-image: url(:/title/img/title/close_icon_hover.png);
}
#close_btn:pressed{
	
	border-image: url(:/title/img/title/close_icon_pressed.png);
}

#close_btn{
	
	border-image: url(:/title/img/title/close_icon_normal.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0">
     <item>
      <widget class="QLabel" name="name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;XX天气&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="change_city">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;城市列表&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QComboBox" name="cities">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="week">
     <property name="text">
      <string>2022年08月31日</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="weather_img">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>天气图片</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="temperature">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;20&lt;/span&gt;&lt;span style=&quot; font-size:26pt; vertical-align:super;&quot;&gt;℃&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="weather">
         <property name="text">
          <string>天气</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="temperature_range">
         <property name="text">
          <string>温度范围</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="humidity">
       <property name="text">
        <string>湿度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="wind_direction">
       <property name="text">
        <string>风向</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ultraviolet_ras">
       <property name="text">
        <string>紫外线</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="air_quality">
       <property name="text">
        <string>空气质量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pm">
       <property name="text">
        <string>pm</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="sunrise_time">
         <property name="text">
          <string>日出时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sunset_time">
         <property name="text">
          <string>日落时间</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../weather.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cities</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>get_weather()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minimize_btn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>-101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>-44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setting_btn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>show_setting_pane()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>-89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pin_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>Form</receiver>
   <slot>pin_main_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>-111</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>get_weather()</slot>
  <slot>show_setting_pane()</slot>
  <slot>pin_main_window()</slot>
 </slots>
</ui>
